ts4.1/index.d.ts

```diff
@@ -234,12 +235,12 @@ export interface TFunction<N extends Namespace = DefaultNamespace, TKPrefix = un
   ): TFuncReturn<N, TKeys, TDefaultResult, TKPrefix>;
 }
 
-export interface TransProps<
+export type TransProps<
   K extends TFuncKey<N, TKPrefix> extends infer A ? A : never,
   N extends Namespace = DefaultNamespace,
   TKPrefix = undefined,
-  E extends Element = HTMLDivElement
-> extends React.HTMLProps<E> {
+  E extends React.HTMLProps<HTMLDivElement> | TextProps = React.HTMLProps<HTMLDivElement>
```

Comment:
Interesting, I've never seen this way to reassign a type, does it really work? Is this the only way? Can't we use type augmentation?